Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine
+MAINTAINER devops-training-bcn@googlegroups.com
+
+ENV binary tcpgoon
+ENV install_path /usr/local/bin
+COPY ${binary} ${install_path}
+
+ENTRYPOINT ["sh", "-c", "${install_path}/${binary}"]
```

Comment:
what "-c" does?

Reply:
this https://explainshell.com/explain?cmd=sh+-c ?

Reply:
Yes. That's some kind of "workaround", as ENTRYPOINT do not accept variables, but this yes. And I'm forced to use a base docker image with sh available (the scratch is still smaller than alpine!) because of this limitation..